iree/test/e2e/linalg_ext_ops/BUILD

```diff
@@ -46,13 +49,33 @@ iree_check_single_backend_test_suite(
             "scan.mlir",
         ],
         include = ["*.mlir"],
+        # linalg_transform_spec is a an MLIR file that specifies a
+        # transformation, it needs to be included as data.
         exclude = [
+            "linalg_transform.mlir",
+            "linalg_transform_spec.mlir",
         ],
     ),
     driver = "dylib",
     target_backend = "dylib-llvm-aot",
 )
 
+iree_lit_test_suite(
+    name = "check_linalg_transform",
+    srcs = ["linalg_transform.mlir"],
+    data = [
+        "//iree/test/e2e/linalg_ext_ops:linalg_transform_spec.mlir",
```

Comment:
If you want to keep the "needs to be included as data" comment it should go here instead